docs/history/changelog.md

```diff
@@ -151,6 +151,7 @@ Per the [normative and non-normative content and changes policy](../governance/n
 * [#1209](https://github.com/open-contracting/standard/pull/1209) Replace "electronic goods" with "digital goods" in the description of 'goods' from the `procurementCategory` codelist, to align with the description in the World Trade Organization's Agreement on Government Procurement, and to avoid confusion between electronic goods like computers and digital goods like software.
 * [#1530](https://github.com/open-contracting/standard/pull/1530) Use consistent wording for "goods, services and/or works" in the `classificationScheme`, `extendedProcurementCategory`, `partyRole` and `procurementCategory` codelists.
 * [#1550](https://github.com/open-contracting/standard/pull/1550) Align and improve code descriptions in `extendedProcurementCategory` and `procurementCategory`.
+* [#1645](https://github.com/open-contracting/standard/pull/1645) Deprecate "parent-to-child" codes. Update the description of 'prior' and add sub-codes. Update the description of 'framework'.
```

Comment:
This needs to mention the codelist name and should name the new codes (and when doing so, the code being deprecated for each can be mentioned explicitly).

Reply:
Done in 37eafe1a681ab9d54487f45d50fa4c4b8ff7b07e. I'm not sure how I missed the codelist name!